src/coreclr/jit/gentree.h

```diff
@@ -2008,7 +2008,6 @@ struct GenTree
 
     void SetContained()
     {
-        assert(IsValue());
```

Comment:
This should not need to remove basic asserts like this. Seems like we should not be containing the `AND` that generates into flags.